test.php

```diff
@@ -10,7 +10,7 @@
  * @link     5
  */
 
-	function a(){ return 2; }
+		function a(){ return 2; }
```

Comment:
:no_entry_sign: **Error**: Opening brace must be the last content on the line (*Generic.Functions.OpeningFunctionBraceKernighanRitchie.ContentAfterBrace*).